charts/hardware-exporter/templates/daemonset.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: {{ include "common.capabilities.deployment.apiVersion" . }}
+kind: DaemonSet
+metadata:
+  name: {{ template "common.names.fullname" . }}
+  namespace: {{ include "common.names.namespace" . | quote }}
```

Comment:
change `common.names.namespace` with `value.namespaceOverride`
change `common.names.fullname` with `value.fullnameOverride`

and etc.